PS8lite.Rmd

```diff
@@ -201,9 +206,17 @@ Fit one Random Forest model `model_rf` using any approach you want.
 Using `model_rf` make predictions and submit to Kaggle.
 
 ```{r}
-
+test <- test %>%
+  mutate(logSalePrice_hat = predict(object = model_rf, newdata = test),
+         SalePrice_hat = exp(logSalePrice_hat) - 1
+         )
+
+submission_rf <- test %>%
+  mutate(SalePrice = SalePrice_hat) %>%
+  select(Id, SalePrice)
+write_csv(submission_rf, "submission_rf.csv")
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.